### PR TITLE
Add psysh.el

### PR DESCRIPTION
**Pull request by zonuexe**
[psysh.el](https://github.com/zonuexe/psysh.el) is PsySH interface on Emacs, and that integrates [php-mode](https://github.com/ejmr/php-mode) and auto-complete.

### CODE REVIEW

**Comment by bobthecow** [comment]
Please alphabetize :)

**Comment by zonuexe** [comment]
@bobthecow OK, fixed.

File: README.md [hunk 1]
```diff
@@ -198,3 +199,4 @@ The PsySH `doc` command is great for documenting source code, but you'll need a
  * Yii 2: [`yii shell`](https://github.com/yiisoft/yii2-shell)
  * Zend Framework 1: [`zf1-psysh`](https://github.com/giorgiolucca/zf1-psysh)
  * Zend Framework 2: [PsyshModule](https://zfmodules.com/gianarb/zf2-psysh-module)
+ * Emacs: [`psysh.el`](https://github.com/zonuexe/psysh.el)
```
Comment (bobthecow) [inline README.md]: Prolly don't need two

Comment (zonuexe) [inline README.md]: I'm sorry...

**Comment by bobthecow** [comment]
Thanks!

**Pull request merged**